src/utils/src/Thread.c

```diff
@@ -198,6 +213,23 @@ PUBLIC_API STATUS defaultCreateThread(PTID pThreadId, startRoutine start, PVOID
     return retStatus;
 }
 
+PUBLIC_API STATUS defaultCreateThread(PTID pThreadId, startRoutine start, PVOID args)
+{
+    STATUS retStatus = STATUS_SUCCESS;
+
+#if defined(KVS_DEFAULT_STACK_SIZE)
+    CHK_STATUS(defaultCreateThreadWithParams(pThreadId, start, KVS_DEFAULT_STACK_SIZE, args));
+#elif defined(CONSTRAINED_DEVICE)
+    CHK_STATUS(defaultCreateThreadWithParams(pThreadId, start, THREAD_STACK_SIZE_ON_CONSTRAINED_DEVICE, args));
+#else
+    CHK_STATUS(defaultCreateThreadWithParams(pThreadId, start, 0, args);
```

Comment:
Missing closing parentheses? Not sure how this wouldn't have failed to build though

Reply:
already fixed